examples/HibernateExample/src/main/java/software/amazon/JpaExample.java

```diff
@@ -16,68 +16,71 @@
 
 package software.amazon;
 
+import org.hibernate.Hibernate;
 import jakarta.persistence.EntityManager;
 import jakarta.persistence.EntityManagerFactory;
 import jakarta.persistence.Persistence;
-import org.hibernate.Hibernate;
 import software.amazon.entity.Address;
 import software.amazon.entity.User;
 
-import java.sql.SQLException;
-
 public class JpaExample {
-    private static EntityManagerFactory entityManagerFactory = null;
 
-    public static void main(String[] args) {
-        try {
-            Address address = new Address();
-            address.setTown("Orangeville");
-            address.setStreet("Faulkner");
-            address.setCountry("Canada");
-            address.setPostal("L1L2M2");
-            insertAddress(address);
-            User user = new User();
-            user.setFirst("Dave");
-            user.setLast("Cramer");
-            user.setAddress(address);
-            insertUser(user);
+  private static final EntityManagerFactory entityManagerFactory = Persistence.createEntityManagerFactory("Atlas");
+
+  public static void main(String[] args) {
+    int i = 0;
+    try {
+      while (true) {
```

Comment:
Need a condition to exit the loop. It could be a time (5 min?) or a number of iteration.